hq/app/schedule/CredentialsNotifier.scala

```diff
@@ -0,0 +1,41 @@
+package schedule
+
+import com.amazonaws.services.sns.AmazonSNSAsync
+import com.gu.anghammarad.Anghammarad
+import com.gu.anghammarad.models.{Email, Notification, Preferred, Target}
+import model.CredentialReportDisplay
+import play.api.Logging
+
+import scala.concurrent.duration.DurationInt
+import scala.concurrent.{Await, ExecutionContext}
+import scala.util.control.NonFatal
+
+
+object CredentialsNotifier extends Logging {
+  val subject = "Action required - old AWS credentials and/or credentials missing MFA"
+  val channel = Preferred(Email)
+  val sourceSystem = "Security HQ Credentials Notifier"
+
+  //TODO write multi-line string
+  // https://github.com/guardian/frontend/blob/9468d4bc3e35dc9213f8f5c18d4b88c70b8667cc/common/app/services/Notification.scala#L71
+  //and write tests for this
+  def createMessage(outdatedKeys: CredentialReportDisplay, missingMfa: CredentialReportDisplay): String = ???
```

Comment:
what's the plan with this?

Reply:
I'll open another PR with this completed soon!